contracts/voting-verifier/src/msg.rs

```diff
@@ -43,6 +43,9 @@ pub struct Poll {
 pub enum QueryMsg {
     #[returns(Poll)]
     GetPoll { poll_id: PollID },
+
+    #[returns(Vec<(String, bool)>)]
```

Comment:
We should probably define this type somewhere since it's used in so many contracts, but not for this PR

Reply:
yes, thinking the same